federated-server/shared-state.js

```diff
@@ -0,0 +1,50 @@
+
+const SharedState = require("shared-state-js");
+const { strMapToObj, objToStrMap } = require('obj2map');
```

Comment:
I'm curious about these two guys, isn't there native support for this functionality?

Reply:
When I tried at the time I didn't find a better way or a native way. That is a library of my authorship that implements the two steps in 15 lines of very basic code.